unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarMeshCombiner/AvatarMeshCombinerUtils.cs

```diff
@@ -42,11 +42,14 @@ public static class AvatarMeshCombinerUtils
         /// </summary>
         /// <param name="layers">A CombineLayer list. You can generate this array using CombineLayerUtils.Slice().</param>
         /// <returns>A list of BoneWeights that share the same skeleton.</returns>
-        public static List<BoneWeight> ComputeBoneWeights( List<CombineLayer> layers )
+        public static BoneWeight[] ComputeBoneWeights( List<CombineLayer> layers )
         {
-            List<BoneWeight> result = new List<BoneWeight>();
             int layersCount = layers.Count;
 
+            int resultSize = 0;
+
+            List<BoneWeight[]> boneWeightArrays = new List<BoneWeight[]>(10);
```

Comment:
where is this `10` coming from?

Reply:
just guessing the max possible wearable parts that might be coming